hedyorg/hedy
# 🪲 Display ask and keypressed modals below the output pane

This PR ensures that the ask modal and the key-pressed modal displayed in the output pane, appear below the output editor instead of on top of it.

Fixes #5946

**How to test**
The output appears in 3 different places and each of them have to be tested:
- output on the /hedy page
- output on the /tryit page
- output in teacher slides (embedded editor) 

I tested with the following code in level 15. Make sure to test consecutive displaying of only the ask and only the key-press modals, and then alternating between the two.
```
for i in range 1 to 10
    print 'Some stuff'
for i in range 1 to 10
    if x is pressed
        print 'ya'
    else
        print 'nee'
    a is ask 'what'
    print a
```

It is essential to check that:
- no modal is displayed on top the output.
- the output window always scrolls to the bottom. It should not be the case that when a modal appears beneath the output, its content gets scrolled up because the latest content will remain hidden (specifically test this for if-pressed).
- check if there are any changes for the turtle canvas
- note that showing the variable list seems to be broken and not related to this change
